datameta/api/submissions.py

```diff
@@ -46,15 +47,17 @@ def validate_submission_access(db, db_files, db_msets, auth_user):
     Raises:
         400 HTTPBadRequest
     """
+    auth_f = authz.file_submission
```

Comment:
I think this is dangerous and not necessarily very readable. Especially using the same short form name for both cases. I'd rather go for a multi-line variant here rather than using a short form name at all (see suggestion below).
```suggestion
```

Reply:
didn't want to touch the long lines, but next time I'll just go for it >:)

Reply:
why is it dangerous, though?